module/src/hooks/calendar/calendar.hooks.ts

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
```

Comment:
should all this be in `components/calendar/calendar.hooks.ts` ?

Reply:
I don't think so no. It's an independent hook that's used by multiple components so it can't really go alongside an existing one.